src/internal/arg.c

```diff
@@ -84,3 +84,98 @@ sexp* rlang_is_missing(sexp* _call, sexp* _op, sexp* args, sexp* env) {
 
   return r_lgl(r_eval(r_x_sym, env) == r_missing_sym);
 }
+
+static sexp* stop_arg_match_call = NULL;
+void arg_match0_abort(const char* msg, sexp* arg_nm_promise, sexp* env);
+sexp* rlang_ext2_arg_match0(sexp* _call, sexp* _op, sexp* args, sexp* env) {
+  args = r_node_cdr(args);
+
+  sexp* arg =            KEEP(r_eval(r_node_car(args), env)); args = r_node_cdr(args);
```

Comment:
We try to avoid column indenting.

Reply:
This was cargo-culted from existing implementation :zany_face: